Sources/ReactiveMoya/ReactiveSwiftMoyaProvider.swift

```diff
@@ -17,6 +18,11 @@ open class ReactiveSwiftMoyaProvider<Target>: MoyaProvider<Target> where Target:
         self.stubScheduler = stubScheduler
         super.init(endpointClosure: endpointClosure, requestClosure: requestClosure, stubClosure: stubClosure, manager: manager, plugins: plugins, trackInflights: trackInflights)
     }
+    
+    fileprivate init(moyaProvider: MoyaProvider<Target>) {
+        self.stubScheduler = nil
```

Comment:
Not quite sure what to do here - right now, using `.reactive` means the `stubScheduler` will always be `nil`. Should this then be a `var` instead of a `let`, or can we solve this in another way?